app/views/adverts/index.html.erb

```diff
@@ -2,7 +2,13 @@
 <% @adverts.each do |advert| %>
 <table>
   <tr>
-    <td width="20%"><div><span class="icon" title="ТОП">TOP</span></div><%= advert.picture %></td>
+    <td width="20%">
+      <% if advert.top %>
+        <div><span class="icon" title="ТОП">TOP</span></div><%= advert.picture %>
```

Comment:
You need to show `picture` anyway, so you can put only TOP icon into `if.. end` block